### PR TITLE
fix: add BLOKKERING_VANWEGE_VERHUIZING initiator indication

### PR DESCRIPTION
**Pull request by hsiliev**
Add BLOKKERING_VANWEGE_VERHUIZING
Fix query for ONDER_CURATELE indication

Solves: PZ-4512

### CODE REVIEW

**Review by edgarvonk — APPROVED** [review]

**Review by infonl-marcel — CHANGES_REQUESTED** [review]
There should be added a language label to the FE; just as and below this list:

`  "indicatie.GEHEIMHOUDING_OP_PERSOONSGEGEVENS": "geheimhouding op persoonsgegevens",
  "indicatie.OVERLEDEN": "overleden",
  "indicatie.OPSCHORTING_BIJHOUDING": "opschorting bijhouden",
  "indicatie.NIET_INGEZETENE": "niet ingezetene",
  "indicatie.ONDER_CURATELE": "onder curatele",
`

**Comment by infonl-marcel** [comment]
I am a bit on doubt if all those changes result in some 'null' support in the frontend; we need to properly check I think

File: src/main/kotlin/net/atos/client/brp/BrpClientService.kt [hunk 1]
```diff
@@ -41,9 +41,17 @@ class BrpClientService @Inject constructor(
         private const val GEBOORTE = "geboorte"
         private const val VERBLIJFPLAATS = "verblijfplaats"
         private const val ADRESSERING = "adressering"
+        private const val INDICATIE_CURATELE_REGISTER = "indicatieCurateleRegister"
 
         private val LOG = Logger.getLogger(BrpClientService::class.java.name)
-        private val FIELDS_PERSOON = listOf(BURGERSERVICENUMMER, GESLACHT, NAAM, GEBOORTE, VERBLIJFPLAATS)
+        private val FIELDS_PERSOON = listOf(
```
Comment (edgarvonk) [inline src/main/kotlin/net/atos/client/brp/BrpClientService.kt:47]: Is this part of this PR? Does not seem to have to do with BLOKKERING_VANWEGE_VERHUIZING?

Better than to merge this PR on the other indicatie Git branch and not on main. Then you only see the relevant changes here.

**Review by infonl-marcel — APPROVED** [review]

File: src/main/kotlin/net/atos/client/brp/BrpClientService.kt (hunk 1 above)
Comment (hsiliev) [reply src/main/kotlin/net/atos/client/brp/BrpClientService.kt:47]: Yes it is. That's a fix for `ONDER_CURATELE` indications

Comment (edgarvonk) [reply src/main/kotlin/net/atos/client/brp/BrpClientService.kt:47]: ah right. sorry, missed that

**Pull request merged**